Directory.Build.props

```diff
@@ -3,6 +3,7 @@
   <Import Project="$(RepositoryEngineeringDir)/apicompat/PublicApiAnalyzer.props" />
 
   <PropertyGroup>
+    <NetCurrent>net8.0</NetCurrent>
```

Comment:
Is this property used anywhere else in arcade or is it solely used by consuming repos? my concern is that changing this in order to avoid having to modify individual projects might unintentionally affect how the infrastructure work.

Reply:
Good point. I can drop this in the cleanup.

Reply:
Looks like efcore calls this `DefaultNetCoreTargetFramework`.

https://github.com/dotnet/efcore/blob/ecfee78eb1fa2b2eaa0dbf945f1d4f8fa571be74/src/EFCore.Abstractions/EFCore.Abstractions.csproj#L5C24-L5C53